server/src/main/java/org/apache/cassandra/sidecar/routes/cdc/Service.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.sidecar.routes.cdc;
+
+import java.util.Locale;
+
+/**
+ * Enum representing various services inside config table in sidecar internal keyspace.
+ */
+public enum Service
```

Comment:
do we really need to have this enum as well as `org.apache.cassandra.sidecar.common.request.ServiceConfig` which are practically the same? can we consolidate?

Reply:
Removed one of them.